### PR TITLE
enh: prod check notion GC workflows are running

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

now that we have 2 "infinite" notion workflows per connector, we must also check that they are running in our production check.

### CODE REVIEW

File: front/production_checks/checks/check_notion_active_workflows.ts [hunk 1]
```diff
@@ -11,13 +12,6 @@ interface NotionConnector {
   workspaceId: string;
 }
 
-export function getWorkflowId(dataSourceInfo: {
```
Comment (flvndvd) [inline front/production_checks/checks/check_notion_active_workflows.ts:14]: 👋

File: front/production_checks/checks/check_notion_active_workflows.ts [hunk 2]
```diff
@@ -31,26 +25,31 @@ async function listAllNotionConnectors() {
...
+    const incrementalSyncHandle: WorkflowHandle = client.workflow.getHandle(
+      getNotionWorkflowId(notionConnector, "never")
+    );
+    const garbageCollectorHandle: WorkflowHandle = client.workflow.getHandle(
+      getNotionWorkflowId(notionConnector, "always")
     );
 
-    const description = await handle.describe();
-    const { status } = description;
+    const descriptions = await Promise.all([
```
Comment (flvndvd) [inline front/production_checks/checks/check_notion_active_workflows.ts:40]: By combining the two checks into a single `Promise.all()`, we somewhat lose track of which specific workflow isn't running...

**Review by flvndvd — APPROVED** [review]
LGTM

File: front/production_checks/checks/check_notion_active_workflows.ts (hunk 2 above)
Comment (fontanierh) [reply front/production_checks/checks/check_notion_active_workflows.ts:40]: Agree, but it's IMO enough to know that one of them isn't running (we filter by connectorId in temporal anyway + the command to restart will restart both workflows anyway)

**Pull request merged**